packages/Discretization/src/DTK_PointSearch_def.hpp

```diff
@@ -596,7 +596,7 @@ void PointSearch<DeviceType>::build_distributor(
             flatten_ranks.push_back( rank_host( i ) );
     }
 
-    typename DeviceType::execution_space space;
+    Kokkos::Serial space;
```

Comment:
It is weird to see a hardcoded `Kokkos::Serial`. Should it be `HostSpace` or something?

Reply:
Why? I want an execution space for my `std::vector`.

Reply:
But isn't it possible to configure Kokkos/Trilinos so that you won't even have `Serial`, and only have `OpenMP`?

Reply:
Should be default host space

Reply:
> But isn't it possible to configure Kokkos/Trilinos so that you won't even have Serial, and only have OpenMP?

I admire you confidence that Trilinos can be build without `Serial` but sure I'll change it.